docs/book/src/roadmap.md

```diff
@@ -0,0 +1,42 @@
+# Cluster API Roadmap
+
+This roadmap is a constant work in progress, subject to frequent revision. Dates are approximations.
+
+## Ongoing
+
+- Documentation improvements
+
+## v0.3 (v1alpha3) ~ March 2020
+
+- Control plane provider concept, KubeadmControlPlane implementation
+- clusterctl v2 - easier installation & upgrades of Cluster API + providers
+- Integration with cloud provider VM scaling groups (MachinePools)
+- Machine failure remediation
```

Comment:
Could add tracking issue for this one
```suggestion
- [Machine failure remediation](https://github.com/kubernetes-sigs/cluster-api/issues/1990)
```

Reply:
I had been omitting links for the in-flight v1alpha3 work, but I'll add them.